### PR TITLE
Issue#57 user info screen

### PR DESCRIPTION
**Pull request by ShrutiJ-01**
Added a screen to ask for username, named it UsernameAddSheet.I have added the styles used for headings in styles/styles.dart. Please let me know about any corrections

### CODE REVIEW

File: lib/styles/style.dart [hunk 1]
```diff
@@ -49,6 +49,35 @@ abstract class Style {
       fontSize: 13.0,
     );
   }
+  ///For button text in UserNameAdd sheet
+  static TextStyle ButtonText(BuildContext context) {
```
Comment (hashirshoaeb) [inline lib/styles/style.dart:53]: rename ButtonText as buttonText

File: lib/screens/UsernameAddSheet.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                child: c.Text('Hello there! ',
+                    style: Style.veryLargeTitle(context)
+                ),
+              ),
+              c.SizedBox(height:10.0),
+              c.Text("So nice to meet you! What do your friends call you?",
+                  style: Style.subTitle(context)
+              ),
+              c.SizedBox(height:h*0.13),
+              c.CupertinoTextField(
```
Comment (hashirshoaeb) [inline lib/screens/UsernameAddSheet.dart:46]: For dark mode, you need to make corrections to make it visible.
Screenshot 2021-01-07 at 12 25 03 AM

You will find ios design resource here https://developer.apple.com/design/resources/

**Comment by hashirshoaeb** [comment]
- Remove 'c.' from all the widgets except for the ones with starting name as Cupertino.
- Rename UsernameAddSheet.dart to username_add_sheet.dart

**Comment by hashirshoaeb** [comment]
You can test dark mode by adding following in CuppertinoApp() in main.dart
```
 theme: c.CupertinoThemeData(brightness: c.Brightness.dark),
```

File: lib/screens/UsernameAddSheet.dart [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  void dispose() {
+    super.dispose();
+    _username.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    double h=c.MediaQuery.of(context).size.height;
+    return c.CupertinoPageScaffold(
+      child: c.SingleChildScrollView(
```
[the remark this answers is not in the document]
Comment (ShrutiJ-01) [reply lib/screens/UsernameAddSheet.dart:29]: I have removed the 'SingleChildScrollView' and set 'resizeToAvoidBottomInset:false' in CupertinoScaffold. This solves the overflow problem. I have adjusted the position of the keyboard relative to the top of the screen such that it doesnot get obscured when the keyboard appears. This is how it looks as of now, should I make any other changes?
star_book_Issue#57

[the remark this answers is not in the document]
Comment (ShrutiJ-01) [reply lib/screens/UsernameAddSheet.dart:29]: After the user is done typing, keyboard can go away and he can see the continue button

[the remark this answers is not in the document]
Comment (hashirshoaeb) [reply lib/screens/UsernameAddSheet.dart:29]: push your changes

**Review by hashirshoaeb — COMMENTED** [review]
Nicely done ✨

**Pull request merged**